src/main/java/core/basesyntax/strategy/operation/BalanceOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+
+public class BalanceOperation implements OperationHandler {
+    private StorageDao storageDao = new StorageDaoImpl();
```

Comment:
Avoid using direct instantiation of `StorageDaoImpl` here. It's better to pass the `StorageDao` instance to the `BalanceOperation` through the constructor, which allows for better testing and flexibility, adhering to the Dependency Inversion Principle.